### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by johnnyomair**
Some parts in our codebase have been deprecated for a very long time. I decided that it's a good time to remove them now. I doubt this will cause too many troubles in existing projects

### CODE REVIEW

File: .changeset/chilly-kids-arrive.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@comet/blocks-admin": minor
```
Comment (thomasdax98) [inline .changeset/chilly-kids-arrive.md:2]: Is removing deprecated functionality a minor change? Sounds more like a major change (but doesn't matter so much as all the changes will be released in v5.0.0 anyway)

**Review by thomasdax98 — DISMISSED** [review]

**Comment by nsams** [comment]
Imho deprecated code doesn't have to be removed just for the sake of removing it. Especially simple renames can be kept for a long time. If there is a reason (eg makes some refactoring simpler) for removing, then yes, of course.

Of course the old names have to be marked as @deprecated, the IDE should strike the old name.

File: packages/admin/admin-rte/src/core/Controls/ControlButton.tsx [hunk 2]
```diff
@@ -11,22 +11,9 @@ export interface IProps {
...
-    selected = false,
-    onButtonClick,
-    icon,
-    children,
-    Icon: deprecatedIcon,
-    classes,
-}: IProps & WithStyles<typeof styles>) {
-    const Icon = icon || deprecatedIcon;
-
+function ControlButton({ disabled = false, selected = false, onButtonClick, icon: Icon, children, classes }: IProps & WithStyles<typeof styles>) {
```
Comment (thomasdax98) [inline packages/admin/admin-rte/src/core/Controls/ControlButton.tsx:16]: Why do we use `icon` lowercase here when we pass a component? Shouldn't it be uppercase for a component and lowercase for an element?

**Comment by johnnyomair** [comment]
> Imho deprecated code doesn't have to be removed just for the sake of removing it.

We should discuss this in today's meeting and document our approach to deprecations 👍🏼

File: .changeset/chilly-kids-arrive.md (hunk 1 above)
Comment (johnnyomair) [reply .changeset/chilly-kids-arrive.md:2]: You're right, removing deprecated code is a major change.

**Comment by johnnyomair** [comment]
Superseded by #1286 and #1287.

As discussed internally: We'll only remove deprecated code once it prevents us from making changes, for instance, a refactoring. Therefore we can close this PR.

**Pull request closed without merging**